src/libsyntax/print/pprust.rs

```diff
@@ -3123,4 +3157,33 @@ mod tests {
         let varstr = variant_to_string(&var);
         assert_eq!(varstr, "principal_skinner");
     }
+
+    #[test]
+    fn test_pretty_print_tokentrees() {
+        use parse;
+        use parse::parse_tts_from_source_str as parse_tts;
+        use pretty::pprust;
+        let orginal = r#"fn main()
+{
+    let program = "+ + * - /";
+    let mut accumulator = 0;
+    . .. ...;
+    a.b;
+    .. .a;
+    a. ..;
+    for token in program.chars()
+    {
+        match token {
+                        '+' => accumulator += 1 , '-' => accumulator -= 1 , '*' => accumulator *= 2 , '/' => accumulator /= 2 , _ => {}
```

Comment:
What should I do here to fix the tidy error? Can't put a newline because I would have another string then.